cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -92,6 +93,7 @@ export enum AnnotationActionTypes {
     GET_JOB = 'GET_JOB',
     GET_JOB_SUCCESS = 'GET_JOB_SUCCESS',
     GET_JOB_FAILED = 'GET_JOB_FAILED',
+    UPDATE_JOB_SUCCESS = 'UPDATE_CURRENT_SUCCESS',
```

Comment:
UPDATE_CURRENT_JOB_FAILED is unused now.
BTW it is still referenced from notifications reducer.